examples/statistics/histogram_demo_cumulative.py

```diff
@@ -1,29 +1,72 @@
 """
-Demo of the histogram (hist) function used to plot a cumulative distribution.
+==========================================================
+Demo of using histograms to plot a cumulative distribution
+==========================================================
+
+This shows how to plot a cumulative, normalized histogram as a
+step function in order to visualize the empirical cumulative
+distribution function (CDF) of a sample. We also use the `mlab`
+module to show the theoretical CDF.
+
+A couple of other options to the ``hist`` function are demonstrated.
+Namely, we use the ``normed`` parameter to normalize the histogram and
+a couple of different options to the `cumulative` parameter. Normalizing
+a histogram means that the bin heights are scaled such that
+the total area is 1. Since we're showing a normalized and
```

Comment:
I still feel there's one sentence missing between here explaining what the `cumulative` argument does, or maybe the "Since..." sentence can go after the next paragraph.